### PR TITLE
Created build-a-blog-generator-with-openai

### PR DESCRIPTION
**Pull request by asiqurrahman**

### CODE REVIEW

**Comment by asiqurrahman** [comment]
Added two gifs

**Comment by asiqurrahman** [comment]
Renamed fille from **build-a-blog-generator-with-openai** to **create-a-blog-generator-with-openai**

File: projects/create-a-blog-generator-with-openai/create-a-blog-generator-with-openai.mdx [hunk 1]
```diff
@@ -0,0 +1,404 @@
+---
+title: Create a Blog Generator with OpenAI
+author: Asiqur Rahman
+datePublished: ?
+description: Learn how to create a blog generator with Python and OpenAI in a few lines of code.
+header: ?
+tags:
+  - ?
```
Comment (sonnynomnom) [inline projects/create-a-blog-generator-with-openai/create-a-blog-generator-with-openai.mdx:8]:
```suggestion
  - intermediate
```

**Comment by Goku-kun** [comment]
I'm merging this in to get this tutorial live and later, we can edit it if there are more edits.

**Pull request merged**